msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -919,6 +969,69 @@ The `extensions.runtimes` property configures the sets of runtimes and actions t
 
 To use `extensions.runtimes`, see [create add-in commands](/office/dev/add-ins/develop/create-addin-commands-unified-manifest), [configure the runtime for a task pane](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-task-pane-command), and [configure the runtime for the function command](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-function-command).
 
+```json
```

Comment:
@erikadoyle this is a draft idea, and not a final representation but the section for the extensions is too lengthy to be placed in the snippet. It makes an already length doc near unbearable to scroll through. The option to split up that section and place it in the body is also not an apt choice, especially after viewing the learn preview site.  The optimal solution would likely be to link to its sample in the [Office Add-ins with the unified app manifest for Microsoft 365]( https://learn.microsoft.com/en-us/office/dev/add-ins/develop/unified-manifest-overview#see-also) doc. Thoughts?